src/main/io/gps.h

```diff
@@ -35,6 +35,152 @@
+    CFG_MSGOUT_NMEA_ID_GSA_UART1 = 0x209100c0, // U1
+    CFG_MSGOUT_NMEA_ID_RMC_I2C = 0x209100ab, // U1
+    CFG_MSGOUT_NMEA_ID_RMC_SPI = 0x209100af, // U1
+    CFG_MSGOUT_NMEA_ID_RMC_UART1 = 0x209100ac, // U1
+    CFG_MSGOUT_UBX_NAV_PVT_I2C = 0x20910006, // U1
+    CFG_MSGOUT_UBX_NAV_PVT_SPI = 0x2091000a, // U1
+    CFG_MSGOUT_UBX_NAV_PVT_UART1 = 0x20910007, // U1
+    CFG_MSGOUT_UBX_NAV_SAT_I2C = 0x20910015, // U1
+    CFG_MSGOUT_UBX_NAV_SAT_SPI = 0x20910019, // U1
+    CFG_MSGOUT_UBX_NAV_SAT_UART1 = 0x20910016, // U1
+    CFG_MSGOUT_UBX_NAV_DOP_I2C = 0x20910038, // U1
+    CFG_MSGOUT_UBX_NAV_DOP_SPI = 0x2091003c, // U1
+    CFG_MSGOUT_UBX_NAV_DOP_UART1 = 0x20910039, // U1
+    CFG_SBAS_USE_TESTMODE = 0x10360002, // L
+    CFG_SBAS_USE_RANGING = 0x10360003, // L
+    CFG_SBAS_USE_DIFFCORR = 0x10360004, // L
+    CFG_SBAS_USE_INTEGRITY = 0x10360005, // L
+    CFG_SBAS_PRNSCANMASK = 0x50360006, // X8
+    CFG_SIGNAL_GPS_ENA = 0x1031001f, // L
+    CFG_SIGNAL_SBAS_ENA = 0x10310020, // L
+    CFG_SIGNAL_GAL_ENA = 0x10310021, // L
+    CFG_SIGNAL_BDS_ENA = 0x10310022, // L
+    CFG_SIGNAL_QZSS_ENA = 0x10310024, // L
+    CFG_SIGNAL_GLO_ENA = 0x10310025, // L
+    CFG_PM_OPERATEMODE = 0x20d00001, // E1
+} ubxValgetsetBytes_e;
+
+/*
+ * replaced by macro, to save space; keeping for reference
+typedef enum {
+    SBAS_SEARCH_ALL = 0,
+    SBAS_SEARCH_PRN120,
+    SBAS_SEARCH_PRN121,
+    SBAS_SEARCH_PRN122,
+    SBAS_SEARCH_PRN123,
+    SBAS_SEARCH_PRN124,
+    SBAS_SEARCH_PRN125,
+    SBAS_SEARCH_PRN126,
+    SBAS_SEARCH_PRN127,
+    SBAS_SEARCH_PRN128,
+    SBAS_SEARCH_PRN129,
+    SBAS_SEARCH_PRN130,
+    SBAS_SEARCH_PRN131,
+    SBAS_SEARCH_PRN132,
+    SBAS_SEARCH_PRN133,
+    SBAS_SEARCH_PRN134,
+    SBAS_SEARCH_PRN135,
+    SBAS_SEARCH_PRN136,
+    SBAS_SEARCH_PRN137,
+    SBAS_SEARCH_PRN138,
+    SBAS_SEARCH_PRN139,
+    SBAS_SEARCH_PRN140,
+    SBAS_SEARCH_PRN141,
+    SBAS_SEARCH_PRN142,
+    SBAS_SEARCH_PRN143,
+    SBAS_SEARCH_PRN144,
+    SBAS_SEARCH_PRN145,
+    SBAS_SEARCH_PRN146,
+    SBAS_SEARCH_PRN147,
+    SBAS_SEARCH_PRN148,
+    SBAS_SEARCH_PRN149,
+    SBAS_SEARCH_PRN150,
+    SBAS_SEARCH_PRN151,
+    SBAS_SEARCH_PRN152,
+    SBAS_SEARCH_PRN153,
+    SBAS_SEARCH_PRN154,
+    SBAS_SEARCH_PRN155,
+    SBAS_SEARCH_PRN156,
+    SBAS_SEARCH_PRN157,
+    SBAS_SEARCH_PRN158,
+} ubxSbasPrnScan_e;
+*/
+
+#define SBAS_SEARCH_ALL    0x0
+#define SBAS_SEARCH_PRN(i) (1 << (i - 120))
+
+typedef enum {
+    UBX_VAL_LAYER_RAM = 0x01,
+    UBX_VAL_LAYER_BBR = 0x02,
+    UBX_VAL_LAYER_FLASH = 0x04,
+} ubloxValLayer_e;
+
+struct ubloxVersion_s {
+    uint32_t hw;
+    const char* str;
+};
+extern struct ubloxVersion_s ubloxVersion_map[];
+#endif
+
+typedef enum {
+    GPS_STATE_UNKNOWN = 0,
+    GPS_STATE_INITIALIZING,
+    GPS_STATE_INITIALIZED,
+    GPS_STATE_CHANGE_BAUD,
+    GPS_STATE_CONFIGURE,
+    GPS_STATE_RECEIVING_DATA,
+    GPS_STATE_LOST_COMMUNICATION,
+    GPS_STATE_COUNT
+} gpsState_e;
```

Comment:
Exposing this enum is not needed. This can go back into `gps.c`
```suggestion
```

Reply:
Those states are currently used in cli.c to show the status of the unit.
This should probably be implemented differently, but that's how it is at the moment, from what I can see

Reply:
I didn't see it. Nevermind then. Then it needs to be exposed in `gps.h` :)